kernel/enter.c

```diff
@@ -62,6 +62,97 @@ long myst_tcall(long n, long params[6])
     return ret;
 }
 
+static int _process_mount_configuration(myst_mounts_config_t* mounts)
```

Comment:
was thinking it was closer to the mount call itself!

Reply:
This is error prone in the future when developers try to add more error checking. They would have to look up the existing checking in two files. There are also performance differences (traverse the list once vs. twice, the call stack for printf etc.), but those are less of concern.

Reply:
I know what you are saying, however the process_mount_configuration is where the config settings are passed into the mount API itself, so it is in fact closer  to where it is used and in my opinion is a more logical place.
The merge code is unlikely to change.

Reply:
Another concern is reducing the size of the kernel as much as possible . I think we should locate as much code as possible outside the kernel. I would do this check inside the TEE but outside the kernel. Maybe in ``myst/tools/enc``. Maybe it can go in ``myst/tools/config.c``.

Reply:
done